howto/anbox/develop-platform.md

```diff
@@ -0,0 +1,84 @@
+Anbox Cloud provides a platform SDK which allows the development of custom platform plugins for the Anbox runtime.
+
+This guide assumes all steps are being run on a machine running Ubuntu 22.04 and hosting the [Anbox Cloud Appliance](https://discourse.ubuntu.com/t/install-appliance/22681).
+
+A platform module must be build on the same version of Ubuntu as the Anbox runtime. This means if you're using one of the Ubuntun 22.04 based Anbox images (e.g. `jammy:android12:arm64`) you must build on Ubuntu 22.04.
+
+In case that you're not running the [Anbox Cloud Appliance](https://discourse.ubuntu.com/t/install-appliance/22681) on an Ubuntu 22.04 system, you can build the platform on a separate system, e.g. in a LXD or docker container or another machine.
+
+To get started you first need to clone the [Anbox Platform SDK](https://github.com/anbox-cloud/anbox-platform-sdk) git repository:
+
+    git clone https://github.com/anbox-cloud/anbox-platform-sdk.git
+
+Before you can build one of the examples, you have to install all necessary build dependencies:
+
+    sudo apt install -y --no-install-recommends cmake build-essential ninja-build dh-exec cmake-extras libgmock-dev libelf-dev
+
+The [Anbox Platform SDK](https://github.com/anbox-cloud/anbox-platform-sdk) comes with various examples which demonstrate different features. The following steps are using the `minimal` example. Alternatively you can choose the `nvidia` example if you're working with NVIDIA GPUs and want to see graphical output.
+
+To build the `minimal` platform example, run the following commands:
+
+    triplet=$(dpkg-architecture -qDEB_BUILD_MULTIARCH)
+    cmake \
+        -G Ninja \
+        -B build \
+        -DCMAKE_PREFIX_PATH=../../lib/anbox-platform-sdk/ \
+        -DCMAKE_INSTALL_LIBDIR=lib/${triplet} \
+        -DCMAKE_INSTALL_PREFIX=/usr examples/minimal/
+    ninja -C build
+
+As a result of the build process you will find a `platform_minimal.so` module in the `build` directory.
+
+[AMS](https://discourse.ubuntu.com/t/about-ams/24321) allows launching Anbox containers in a special development mode which is meant for development tasks, specifically for addon or platform development. The Anbox runtime is instructed, when in development mode, not to terminate the container when it detects failures or other problems.
```

Comment:
Dev mode is documented here: https://anbox-cloud.io/docs/exp/containers#dev-mode

Reply:
Added

Reply:
Added